AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1679,6 +1679,7 @@ gizmodo.com,kotaku.com###newsletterPromoModule
 cnet.com,newyorker.com##.newsletter-subscribe-form
 newyorker.com#?#div[class^="PageSections__container"] > section[class^="SoloPromoSection__promoWrapper"]:has(> div[class^="SoloPromoSection__promoColumn"] > p[class^="SoloPromoSection__promo"] > span > a[href="https://www.newyorker.com/newsletters"])
 belretail.by##.bannerAfterArticle
+ettoday.net#?#.inner > div.block:has(> .sidebar-hot-express)
```

Comment:
Don't forget about tags after `>`

Reply:
You mean as in `div.sidebar-hot-express`?

Reply:
Yes

Reply:
oh ok
though we dont need em  in for a short selectors like this one